packages/pharos/src/components/toast/PharosToast.react.stories.jsx

```diff
@@ -30,7 +30,7 @@ export const Base = {
           const event = new CustomEvent('pharos-toast-open', {
             detail: {
               content:
-                'The item has moved to your <pharos-link href="#" on-background bold>Workspace</pharos-link>.',
+                'The item has moved to your <PharosLink href="#" on-background bold>Workspace</PharosLink>.',
```

Comment:
Does `PharosLink` need to be imported like the other React Pharos components? If so, I think the import is missing.

Reply:
I'm sort of wondering if links in toasts work at all in React, now...